Lib/tkinter/test/test_tkinter/test_misc.py

```diff
@@ -48,6 +48,39 @@ def test_tk_setPalette(self):
                 '^must specify a background color$',
                 root.tk_setPalette, highlightColor='blue')
 
+    def test_after_cancel(self):
+        root = self.root
+        timer1 = root.after(5000, lambda: 'break')
```

Comment:
Make commands having a side effect (for example adding an item to the list) and check that after canceling events and calling `update()` they are not executed.

Reply:
I made a counter within a callback function.  Is that OK or is a list a better solution?